web/src/containers/StateDashboard.js

```diff
@@ -33,63 +33,75 @@ const StateDashboard = (
-                <Button
-                  variation="primary"
-                  size="small"
-                  className="right inline-block"
-                  onClick={create}
+        <div className="site-main ds-l-col--8 ds-u-margin-x--auto ">
+          <h1 className="ds-h1 ds-u-margin-top--3">
+            {t('stateDashboard.title', { state: state.name })}
+          </h1>
+          <Instruction source="stateDashboard.instruction" />
+
+          <div className="ds-u-border-bottom--2 ds-u-margin-bottom--1 ds-u-margin-top--6 ds-u-padding-bottom--1">
+            <Button
+              variation="primary"
+              className="ds-u-float--right"
+              onClick={create}
+            >
+              Create new&nbsp;&nbsp;
+              <Icon icon={faPlusCircle} />
+            </Button>
+            <h2 className="h2">{state.name} APDs</h2>
+          </div>
+
+          {fetching ? <Loading /> : null}
+          {!fetching && apds.length === 0 ? t('stateDashboard.none') : null}
+          <div className="ds-l-container">
+            {apds.map(apd => (
+              <div
+                key={apd.id}
+                className="ds-l-row ds-u-border-bottom--2 ds-u-margin-bottom--2 ds-u-padding-bottom--2"
+              >
+                <div
+                  className="ds-l-col--1 ds-u-valign--middle ds-u-text-align--center"
+                  style={{ alignSelf: 'center' }}
                 >
-                  Create new&nbsp;&nbsp;
-                  <Icon icon={faPlusCircle} />
-                </Button>
-              </div>
-              <div className="p3">
-                {fetching ? <Loading /> : null}
-                {!fetching && apds.length === 0
-                  ? t('stateDashboard.none')
-                  : null}
-                {apds.map(apd => (
-                  <div key={apd.id} className="p2 mb2 bg-gray-lightest">
-                    <div className="inline-block p2 mr2 bg-white blue rounded left">
-                      <img
-                        src="/static/img/icon-document.svg"
-                        alt=""
-                        className="align-middle"
-                        style={{ minWidth: '33px' }}
-                      />
-                    </div>
-                    <h3 className="inline-block">
-                      <a href="#!" onClick={open(apd.id)}>
-                        HITECH APD for FFY {apd.years.join(', ')}
-                      </a>
-                    </h3>
-                    <Button
-                      variation="danger"
-                      size="small"
-                      className="right"
-                      onClick={delApd(apd)}
-                    >
-                      Delete
-                    </Button>
-                  </div>
-                ))}
+                  <span
+                    className="ds-u-fill--primary-alt-lightest ds-u-padding--2"
+                    style={{ marginLeft: '-16px' }}
```

Comment:
Unclear to me why this was necessary, but this is the width of the icon and this pulls the span back so that it's actually centered. I don't know why it isn't centered anyway because the containing `div` has a `ds-u-text-align--center` and :all-the-shruggies:

Reply:
I'm working on a patch PR for ya (because explaining "take this class from here and put it here" just doesn't work so well with the commit suggestions). This is a good question, though! The columns in nested rows are smaller than the columns in their parents, which is an eternal pain point between design and front end dev. If you look at the original screenshot, you'll see that this icon is supposed to be "one column" wide. Except now we're inside a container inside a container, so this row makes 12 columns out of what used to be 8. (You're accounting for this numerically already.) One column is now too small to actually hold this icon at size, so it squeezes it too far to the right. Because it's a `span`, it'll overflow the column and do its own thing. Here's a screenshot that makes it easier to see what's happening, with `span` set to `display: block`:
<img width="501" alt="Screen Shot 2019-05-17 at 10 39 30 AM" src="https://user-images.githubusercontent.com/509309/57942981-1c7ba600-7890-11e9-9883-c5f38621ebee.png">

CSS is the worst.

Reply:
Oh, this is also a prime example of why a designer may want to make a static visual element a column wide, but implementing it so that it actually is a column width gets messy :-/